tamr_unify_client/client.py

```diff
@@ -145,6 +145,22 @@ def datasets(self):
         """
         return self._datasets
 
+    def create_project(unify, project_config):
```

Comment:
I just noticed that this is probably here because you made changes on top of your other changes for #134 . Let's chat after standup about how to isolate your PRs from one another.